modules/localstack/src/test/java/org/testcontainers/containers/localstack/SimpleLocalstackTest.java

```diff
@@ -47,6 +56,17 @@ public void s3Test() throws IOException {
         final S3Object object = s3.getObject("foo", "bar");
         final String content = IOUtils.toString(object.getObjectContent(), Charset.forName("UTF-8"));
         assertEquals("The object can be retrieved", "baz", content);
+    }
+
+    @Test
+    public void hostnameSqsTest() {
+        AmazonSQS sqs = AmazonSQSClientBuilder.standard()
+            .withEndpointConfiguration(localstack.getEndpointConfiguration(SQS))
+            .withCredentials(localstack.getDefaultCredentialsProvider())
+            .build();
 
+        CreateQueueResult queueResult = sqs.createQueue("baz");
+        String fooQueueUrl = queueResult.getQueueUrl();
+        assertThat("Created queue has external hostname URL", fooQueueUrl, containsString(TEST_HOSTNAME));
```

Comment:
not critical, but would be better to parse the returned URL and compare the host, otherwise result like "foobar.localhost" will pass too, but fail in real world

Reply:
Now comparing with the full URL; still using the slightly simplistic string contains mechanism though.